src/main/java/com/epam/edumanagementsystem/student/rest/service/StudentService.java

```diff
@@ -36,4 +36,6 @@ public interface StudentService {
     StudentDto updateForPic(StudentDto studentDto);
 
     List<StudentDto> findStudentsByClassName(String name);
+
+    boolean existStudentById(Long id);
```

Comment:
Rename to existsById. No need to specify the student as the StudentService class name